.github/workflows/docker.build.yml

```diff
@@ -3,9 +3,7 @@ name: Docker build
 on:
   push:
     branches:
-      - 'main'
-    tags:
```

Comment:
I deleted this by accident, but I'm not sure if this is even needed in the current workflow?

Reply:
Yeah, we probably are now safely away from the "bothering to tag" workflow